app/controllers/exams_controller.rb

```diff
@@ -0,0 +1,28 @@
+class ExamsController < ApplicationController
+	before_action :find_exam, only: [:show]
+	def show
+		# @course_id = params[:course_id]
+		# @course = @exam.course
+		@numbers = current_user.exams.find_by(course_id: params[:course_id]).number_question
+		@questions = @exam.questions
+		@question = @questions.find_by id: params[:id]
+		@question_id = params[:id]
+			if @question_id != nil
```

Comment:
```suggestion
			if @question_id.present?
```
mà có cần thiết phải dùng biến instance k anh ơi ?
anh có truyền biến này đi đâu kjhoong?